oggm/tests/test_models.py

```diff
@@ -2658,3 +2658,97 @@ def test_elevation_feedback(self):
             plt.xlabel('Vol (km3)')
             plt.legend()
             plt.show()
+
+
+class TestMergedHEF(unittest.TestCase):
+
+    def setUp(self):
+        # test directory
+        self.testdir = os.path.join(get_test_dir(), 'tmp_merged')
+        if not os.path.exists(self.testdir):
+            os.makedirs(self.testdir)
+
+        # Init
+        cfg.initialize()
+        cfg.set_intersects_db(get_demo_file('rgi_intersect_oetztal.shp'))
+        cfg.PATHS['dem_file'] = get_demo_file('srtm_oetztal.tif')
+        cfg.PATHS['climate_file'] = get_demo_file('histalp_merged_hef.nc')
+        cfg.PARAMS['correct_for_neg_flux'] = True
+        cfg.PARAMS['baseline_climate'] = 'CUSTOM'
+        cfg.PATHS['working_dir'] = self.testdir
+        cfg.PARAMS['border'] = 100
+        cfg.PARAMS['prcp_scaling_factor'] = 1.75
+        cfg.PARAMS['temp_melt'] = -1.75
+
+    def tearDown(self):
+        self.rm_dir()
+
+    def rm_dir(self):
+        shutil.rmtree(self.testdir)
+
+    def clean_dir(self):
+        shutil.rmtree(self.testdir)
+        os.makedirs(self.testdir)
+
+    @pytest.mark.slow
+    def test_merged_simulation(self):
+        import geopandas as gpd
+
+        hef_file = utils.get_demo_file('rgi_oetztal.shp')
+        rgidf = gpd.read_file(hef_file)
+
+        # Get HEF and Kesselwand
+        glcdf = rgidf.loc[(rgidf.RGIId == 'RGI50-11.00897') |
+                          (rgidf.RGIId == 'RGI50-11.00787')].copy()
+        gdirs = workflow.init_glacier_regions(glcdf)
+        workflow.gis_prepro_tasks(gdirs)
+        workflow.climate_tasks(gdirs)
+        workflow.inversion_tasks(gdirs)
+        workflow.execute_entity_task(tasks.init_present_time_glacier, gdirs)
+
+        # run parameters
+        years = 200  # arbitrary
+        y0 = 1950  # arbitrary
+        tbias = -1.0  # arbitrary
+        mbbias = 0.0  # necessary?
```

Comment:
necessary only if you want the MB at t* , i.e. a balanced MB

Reply:
When I wrote this, I wasn't sure how to handle the MB calibration bias and have since then forgotten about it....
I now implemented the `local_mustar` files to be copied to the merged GDir. MultipleFlowlineMB can now use a different bias for each flowline if they originate from different glaciers.